lib/screens/user/add_edit.dart

```diff
@@ -43,7 +43,9 @@ class _UserAddEditState extends State<UserAddEdit> {
     return Scaffold(
       appBar: AppBar(
         title: Text(
-          widget.isEdit ? 'Edit user' : 'Add new user',
+          widget.isEdit
+              ? ShipantherLocalizations.of(context).editParam('user')
```

Comment:
All the text values should be translated

Reply:
Also for strings that have plural form, you might want to use something similar to https://github.com/bigpanther/shipanther/commit/876b8e5243624fa0dbaae279fcfb963c250ba9a1. It would avoid a lot of repetition as a value of 0,1,or >1 can be passed to the translation method.